azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -86,12 +98,115 @@ public ProjectManager(ProjectLoader loader, Props props) {
     new XmlValidatorManager(prop);
     loadAllProjects();
     loadProjectWhiteList();
+
+    CleanerThread cleanerThread = new CleanerThread();
+    cleanerThread.start();
+  }
+
+  /**
+   * Cleaner thread to clean projects. Runs every day.
+  **/
+  private class CleanerThread extends Thread {
+
+    // runs every day
+    private static final long CLEANER_THREAD_WAIT_INTERVAL_MS = 24 * 60 * 60 * 1000;
+    private static final String DELETER = "azkaban";
```

Comment:
Instead can't we simply have a private static final User deleter = new User(DELETER); ?

Reply:
ack